src/Components/Classes/Form/Form.jsx

```diff
@@ -0,0 +1,135 @@
+  };
+
+  return (
+    <>
+      <button
+        className={styles.buttonSwitch}
+        onClick={() => {
+          setFormCreateSwitch(!formCreateSwitch);
+          setError(false);
+        }}
+      >
+        + Add New
+      </button>
+      <div className={styles.formContainer}>
+        {formCreateSwitch && activities.length !== 0 && trainers.length !== 0 ? (
+          <form onSubmit={onSubmit}>
+            <div className={styles.subContainer}>
+              <div className={styles.inputContainer}>
+                <label>Activity</label>
+                <select value={classes.activity} name="activity" onChange={onChangeInput}>
+                  <option value="">Select an activity</option>
+                  {activities.map((activity) => (
+                    <option key={activity._id} value={activity._id}>
+                      {activity.name}
+                    </option>
+                  ))}
+                </select>
+                {error && classes.activity === '' ? (
+                  <span className={styles.error}>Field is required</span>
+                ) : null}
+              </div>
+              <div className={styles.inputContainer}>
+                <label>Trainer</label>
+                <select value={classes.trainer} name="trainer" onChange={onChangeInput}>
+                  <option value="">Select a trainer</option>
+                  {trainers.map((trainer) => (
+                    <option key={trainer._id} value={trainer._id}>
+                      {trainer.firstName}
+                    </option>
+                  ))}
+                </select>
+                {error && classes.trainer === '' ? (
+                  <span className={styles.error}>Field is required</span>
+                ) : null}
+              </div>
+              <div className={styles.inputContainer}>
+                <label>Day</label>
+                <select value={classes.day} name="day" onChange={onChangeInput}>
+                  <option value="">Select a day</option>
+                  <option value="Monday">Monday</option>
+                  <option value="Tuesday">Tuesday</option>
+                  <option value="Wednesday">Wednesday</option>
+                  <option value="Thursday">Thursday</option>
+                  <option value="Friday">Friday</option>
+                  <option value="Saturday">Saturday</option>
+                  <option value="Sunday">Sunday</option>
+                </select>
+                {error && classes.day === '' ? (
+                  <span className={styles.error}>Field is required</span>
+                ) : null}
+              </div>
+              <div className={styles.inputContainer}>
+                <label>Time</label>
+                <input type="time" value={classes.time} name="time" onChange={onChangeInput} />
+                {error && classes.time === '' ? (
+                  <span className={styles.error}>Field is required</span>
+                ) : null}
```

Comment:
can include an error condition where "classes.time === 0 "  to prevent making a server call.